### ISSUE TITLE
Feature request: Alt Text on Videos

### ISSUE THREAD

**Issue by jeffsikes** [Dimillian/IceCubesApp#1710]
[x] Search that this request don't already exist before creating it.

Please add alt text for uploaded videos. The API endpoints for it are the same as if you were adding images. I know you're adding some AI bot integration for images, but I don't think that would work for videos, something to take into consideration as well.

I don't know if Ice Cubes supports uploading audio files, but if so, that should include alt text options as well.

Thanks!

**Comment by Lr-K** [comment]
I have tested this on macOS and iOS this functionality already exists. What is the case where it does not?

**Comment by cabbey** [comment]
> I have tested this on macOS and iOS this functionality already exists. What is the case where it does not?

Can you link to a video you posted with Ice Cubes and alt text where the alt text actually came through on the server?

While the UI allows a user to enter alt text, it does not appear to be posted. (I think this may actually have been more clear if it had been filed as a bug than a feature request.)

**Comment by jeffsikes** [comment]
Alt text wasn't an option on videos at the time I submitted the request. Now, it does. Great! 

https://mastodon.social/@box464/111900894406598275

Some issues still exist but could be considered separate items.

The AI generative alt text doesn't work for videos, but it still appears and lets you try, failing silently. This should be removed as an option if the upload is not an image. Fails on animated gifs, too.

Audio uploads don't seem possible, at least on the iPhone I'm on right now, but that is another issue. Similar efforts as the AI generative bug listed tho, once audio file types are allowed.
